### ISSUE TITLE
youtube music update

### ISSUE THREAD

**Issue by mailinglists35** [lawfx/YoutubeNonStop#34]
Hi,

I think youtube music found it's way to pause the music.
would you mind looking at it again?

thanks!

**Comment by Amariithynar** [comment]
Seconding this. Youtube music is constantly pausing again and *staying* paused, regardless of which tab is active. Very vexing.

**Comment by mailinglists35** [comment]
it's actually weird, it only happened once. now it's back to normal (normal = "Video paused. continue wathing?" but playback continues)

**Comment by Amariithynar** [comment]
It's still happening for me. Are you leaving the tab on the tab the music is playing in?

**Comment by mailinglists35** [comment]
nope, I actually use a dedicated browser for it - firefox nightly - with only music.youtube.com open

**Comment by Amariithynar** [comment]
That could be why then. I am not part of the Nightly branch, nor do I leave a dedicated window open specifically for it- however even when I have left it open to that tab, it still does not work, for me.

**Comment by anoadragon453** [comment]
Also on Firefox nightly and getting pauses.
